packages/interactivity/src/proxies/test/state-proxy.ts

```diff
@@ -0,0 +1,1187 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable @typescript-eslint/no-shadow */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+/**
+ * External dependencies
+ */
+import { effect } from '@preact/signals-core';
+/**
+ * Internal dependencies
+ */
+import { proxifyState, peek } from '../';
+import {
+	setScope,
+	resetScope,
+	setNamespace,
+	resetNamespace,
+	getContext,
+	getElement,
+} from '../../hooks';
+
+type State = {
+	a?: number;
+	nested: { b?: number };
+	array: ( number | State[ 'nested' ] )[];
+};
+
+const withScopeAndNs = ( scope, ns, callback ) => () => {
+	setScope( scope );
+	setNamespace( ns );
+	try {
+		return callback();
+	} finally {
+		resetNamespace();
+		resetScope();
+	}
+};
+
+describe( 'interactivity api - state proxy', () => {
```

Comment:
Same here?


```suggestion
describe( 'interactivity api', () => {
	describe( 'state proxy', () => {
```

And we should also change it in the tests of the `utils` file.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/62734/commits/75d088908bd5e8782df88b61215151ea1e43de4f.